#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
bool emergency_stop
uint8 front_steer_angle
uint8 rear_steer_angle
